src/webui/www/private/scripts/client.js

```diff
@@ -781,8 +781,26 @@ window.addEvent('load', function() {
         addMainWindowTabsEventListener();
         addSearchPanel();
     }
+
+    registerMagnetHandler();
+    handleMagnetParam();
 });
 
+function registerMagnetHandler() {
+    const urlObject = new URL(location.href);
+    urlObject.searchParams.set('magnet', '');
```

Comment:
Can you test it in IE11? we still support it currently.
IE11 doesn't support `URL.search​Params` but at least this line shouldn't break the whole WebUI.

Reply:
Sure. I'll refactor it to eliminate the dependence on URLSearchParams and will test it in IE11.